lstchain/reco/dl0_to_dl1.py

```diff
@@ -334,8 +334,16 @@ def r0_to_dl1(input_filename=get_dataset_path('gamma_test_large.simtel.gz'),
                         dl1_container.ucts_time = ucts_time
                         dl1_container.dragon_time = dragon_time
 
-                        if pointing_file_path and dragon_time > 0:
-                            azimuth, altitude = pointings.cal_pointingposition(dragon_time, drive_data)
+                        # Select the timestamps to be used for pointing interpolation
+                        if config['timestamps_pointing'] == "ucts":
+                            event_timestamps = ucts_time
+                        elif config['timestamps_pointing'] == "dragon":
+                            event_timestamps = dragon_time
+                        elif config['timestamps_pointing'] == "tib":
+                            event_timestamps = tib_time
+
```

Comment:
```
else:
    raise ValueError("the timestamps_pointing option is not a valid one")
```

Reply:
Thanks, @vuillaut!
The error should be raised whenever there is more than one timestamp option set as true, right? So first we have to check this.

We were thinking of rather creating a timestamps_pointing dictionary within config file. It would look like this:

```
"timestamps_pointing":{
    "ucts": true
    "dragon": false
    "tib": false
}
```
and check whether there is more than one true element with something like this:
`
list(filter(config["timestamps_pointing"].get, config["timestamps_pointing"]))
`
So it would be:

```
if len(list(filter(config["timestamps_pointing"].get,
                   config["timestamps_pointing"]))) > 1:
    raise ValueError("the timestamps_pointing option is not a valid one")
```

But it's quite the opposite of explicit. What do you think?

Reply:
> ```
> else:
>     raise ValueError("the timestamps_pointing option is not a valid one")
> ```

OK, I didn't quite understand yesterday when I saw it first. Now I also prefer the first implementation with the addition of these (else & raise) lines. Makes the code clearer. With the second implementation, it wouldn't be so clear how to raise the error, because it has to be checked whether there is only one true option or not.